gcode/tests/smoke_test.rs

```diff
@@ -54,15 +36,6 @@ fn expected_program_2_output() {
     assert_eq!(got.len(), 20);
     // check lines without any comments
     assert_eq!(got.iter().filter(|l| l.comments().is_empty()).count(), 11);
-
-    let gcodes: Vec<_> = got.iter().flat_map(|l| l.gcodes()).cloned().collect();
-    let expected = vec![
-        GCode::new(Mnemonic::ToolChange, 2.0, Span::PLACEHOLDER),
-        GCode::new(Mnemonic::Miscellaneous, 3.0, Span::PLACEHOLDER)
-            .with_argument(Word::new('S', 447.0, Span::PLACEHOLDER))
-            .with_argument(Word::new('F', 80.0, Span::PLACEHOLDER)),
-    ];
-    pretty_assertions::assert_eq!(gcodes, expected);
```

Comment:
I'd prefer if we kept these assertions in. That way we aren't just checking that there were 20 lines and 11 lines without comments, we're also checking we got the exact gcodes we were expecting.

Reply:
This test fails because these `expected` g-codes aren't the full picture. For some odd reason, the test only checks the first line of the original g-code, and the test fails because everything that comes after it doesn't match. This is an example that would benefit from your suggestion to make these types of tests more readable, so we can test the entire file without a ton of verbiage.

This test was originally ignored. I figured it was better to test _something_ than to ignore it entirely.